### ISSUE TITLE
"How to report bugs" link in readme

### ISSUE THREAD

**Issue by nmatt** [apache/netbeans#6767]
### Apache NetBeans version

Apache NetBeans latest daily build

### What happened

The "[How to report bugs](https://github.com/apache/netbeans#reporting-bugs)" link in the readme is a 404: https://netbeans.apache.org/participate/report-issue.html

In addition, it's not clear to me if new issues should only be submitted here on GitHub, or if the Apache Jira is also still fine, given that https://issues.apache.org/jira/projects/NETBEANS/issues/ still exists.

### Did this work correctly in an earlier version?

No / Don't know

### Operating System

n/a

### JDK

n/a

### Apache NetBeans packaging

Other

### Are you willing to submit a pull request?

No

**Comment by ebarboni** [comment]
This is fixed by htacces changes

**Comment by nmatt** [comment]
@mbien: This still produces a 404 for me.

**Comment by mbien** [comment]
@nmatt yeah I have a similar issue, when I open it with firefox (which is my regular browser) i see 404 but when I open it with a fresh user session in chrome it works fine.

Lets wait a little bit, maybe it fixes itself, otherwise we have to investigate what is causing this. The redirect is in place as you can see in the linked issue.

**Comment by mbien** [comment]
here some more info about 301redirect caches and firefox: https://superuser.com/questions/467999/clear-301-redirect-cache-in-firefox

so I set `browser.cache.check_doc_frequency` to 1, clicked the link which now redirected correctly, then I set it to 3 again which is the default value. I think it is essentially a firefox bug, since FF should in my opinion not store redirects permanently if they lead to 404 - even if it is a permanent redirect.

closing again :)

**Comment by nmatt** [comment]
@mbien: Thanks for the investigation! The F12 method worked for me.
